mirgecom/symbolic.py

```diff
@@ -0,0 +1,104 @@
+__copyright__ = """Copyright (C) 2020 CEESD Developers"""
```

Comment:
```suggestion
__copyright__ = """Copyright (C) 2020 University of Illinois Board of Trustees"""
```

https://otm.illinois.edu/disclose-protect/open-source-licensing